examples/tree_2d_dgsem/elixir_diffusion_steady_state_linear_map.jl

```diff
@@ -0,0 +1,107 @@
+using Trixi
+
+###############################################################################
+
+# Build pure diffusion (Laplace) operator
+advection_velocity = (0, 0)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+diffusivity() = 1
+equations_parabolic = LaplaceDiffusion2D(diffusivity(), equations)
+
+# The hyperbolic flux does not matter for this example since
+# the hyperbolic part is zero.
+solver = DGSEM(polydeg = 5, surface_flux = flux_central)
+
+coordinates_min = (0.0, 0.0)
+coordinates_max = (1.0, 1.0)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level = 2,
+                n_cells_max = 80_000,
+                periodicity = false)
+
+# Analytical/continuous steady-state solution
```

Comment:
Is there a reference for this analytical solution or did you develop it yourself?